src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const numbersArray = [...document.querySelectorAll('.population')];
```

Comment:
Ensure that the elements with the class 'population' contain valid numerical values. If any element contains non-numeric text, `parseFloat` will return NaN, which could affect the total and average calculations.